app/controllers/jobs_controller.rb

```diff
@@ -0,0 +1,68 @@
+class JobsController < ApplicationController
+  before_action :set_job, only: [:show, :edit, :update, :destroy]
+  before_action :set_company, only: [:new, :create, :show, :edit, :update, :destroy]
+
+  def index
+    if params.include?("sort")
+      @jobs = Job.all.order(:level_of_interest).reverse
+      render :interest
+    else
+      @company = Company.find(params[:company_id])
+      @jobs = @company.jobs
+      @contact = Contact.new
+      @contacts = @company.contacts.all.order('created_at DESC')
+    end
+  end
+
+  def new
+    @job = Job.new()
+    @category_options = Category.all.map{ |c| [c.title, c.id ]}
+  end
+
+  def create
+    @job = @company.jobs.new(job_params)
+    if @job.save
+      flash[:success] = "You created #{@job.title} at #{@company.name}"
+      redirect_to company_job_path(@company, @job)
+    else
+      render :new
+    end
+  end
+
+  def show 
+    @comment = Comment.new
+    @comments = @job.comments.all.order('created_at DESC')
+  end
+
+  def edit
+    @category_options = Category.all.map{ |c| [c.title, c.id ]}
+  end
+
+  def update
+    @category_options = Category.all.map{ |c| [c.title, c.id ]}
```

Comment:
same here. if this was in a view helper then we wouldn't need to write it twice, we could just call it in our views!